### PR TITLE
Provide Yum for non-debian builds

### PR DESCRIPTION
**Pull request by alexmorley**
ref #118

### CODE REVIEW

**Comment by tkelman** [comment]
Does it need the -devel package or is the library alone enough?

**Comment by alexmorley** [comment]
I needed -devel ...

**Comment by nalimilan** [comment]
It shouldn't be needed if you add `libzmq.so.4` and `libzmq.so.5` to the list of aliases.

File: deps/build.jl [hunk 1]
```diff
@@ -16,12 +16,13 @@ function validate(name, handle)
...
 end
 
-zmq = library_dependency("zmq", aliases = ["libzmq", "libzmq.so.3"], validate = validate)
+zmq = library_dependency("zmq", aliases = ["libzmq", "libzmq.so.3", "libzmq.so.4", "libzmq.so.5"], validate = validate)
 
 provides(Sources, URI("https://archive.org/download/zeromq_3.2.4/zeromq-3.2.4.tar.gz"), zmq)
 provides(BuildProcess, Autotools(libtarget = "src/.libs/libzmq." * BinDeps.shlib_ext), zmq)
 
 provides(AptGet, "libzmq3", zmq, os = :Linux)
+provides(Yum, "czmq-devel", zmq, os = :Linux)
```
Comment (nalimilan) [inline deps/build.jl]: You don't need `-devel` then, right?

Comment (alexmorley) [inline deps/build.jl]: nope...

**Comment by nalimilan** [comment]
Thanks!

**Pull request merged**